retrieval/discovery/nerve.go

```diff
@@ -139,5 +140,9 @@ func parseNerveMember(data []byte, path string) (*model.LabelSet, error) {
 	labels[nerveEndpointLabelPrefix+"_port"] = model.LabelValue(fmt.Sprintf("%d", member.Port))
 	labels[nerveEndpointLabelPrefix+"_name"] = model.LabelValue(member.Name)
 
+	for k, v := range member.Labels {
+		labels[nerveEndpointLabelPrefix+"_labels_"+k] = model.LabelValue(v)
```

Comment:
Should also be singular _label_ to be consistent with kubernetes and
marathon implementations.

On Wed, Aug 24, 2016 at 4:36 PM Brian Brazil <notifications@github.com>
wrote:

> In retrieval/discovery/nerve.go
> <https://github.com/prometheus/prometheus/pull/1916#discussion_r76068546>:
>
> > @@ -139,5 +140,9 @@ func parseNerveMember(data []byte, path string) (*model.LabelSet, error) {
> >  	labels[nerveEndpointLabelPrefix+"_port"] = model.LabelValue(fmt.Sprintf("%d", member.Port))
> >  	labels[nerveEndpointLabelPrefix+"_name"] = model.LabelValue(member.Name)
> >
> > +	for k, v := range member.Labels {
> > +		labels[nerveEndpointLabelPrefix+"_labels_"+k] = model.LabelValue(v)
>
> k need sanatizing
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/prometheus/prometheus/pull/1916/files/ab2607a3c91c988ba6c3888c0ebacd826afb70cb#r76068546>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAANaBNeacjsjq1CQzPDt4Y9uMH2edKFks5qjFbpgaJpZM4JsC33>
> .
>